saya/core/src/prover/atlantic/client.rs

```diff
@@ -178,6 +179,49 @@ struct AtlanticQueryJobsResponse {
     jobs: Vec<AtlanticQueryJob>,
 }
 
+#[derive(Debug, Clone)]
+pub struct MetadataUrls {
+    //Currently we dont use metadata,input and program, but it might be worth to add them in the
+    //future to get more information about the proof/trace in case of failure
+    pub metadata: Option<String>,
```

Comment:
```suggestion
    /// Currently we dont use metadata,input and program, but it might be worth to add them in the
    /// future to get more information about the proof/trace in case of failure
    pub metadata: Option<String>,
```